Rubberduck.Deployment.Build/RubberduckPostBuildTask.cs

```diff
@@ -240,11 +240,40 @@ private void CreateIdlFile(DllFileParameters parameters)
 
         private void CompileWithMidl(DllFileParameters parameters, string batchPath)
         {
-            var targetPath = TargetDir.TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
+            var targetPath = Path.Combine(Path.GetTempPath(), "RubberduckMidl");
+            if (Directory.Exists(targetPath))
+            {
+                Directory.Delete(targetPath, true);
```

Comment:
Shouldn't this still be in a `try`, even with the `if` test? There's still a possibility of a race condition, and if I'm reading this correctly (which I may not), wouldn't anything important in `targetPath` would just get overwritten if the `Delete` failed?

Reply:
Not sure there can be a race condition within a build task. MSBuild tasks are meant to be processed sequentially. Besides, I want it to throw (and fail the build) if there's a problem with the IO. Ain't nothing gonna get built with nothing!

Reply:
There can always be a race condition with IO. Our process isn't the only racer, but if clearing the directory is legitimately a failure condition then we're all good.